src/libs/auth.ts

```diff
@@ -270,34 +289,36 @@ export const loadAuthToken = async (includeBillingScope = false, popUp = false):
 
 const tryLoadAuthToken = async (includeBillingScope = false, popUp = false): Promise<AuthTokenState> => {
   const args: ExtraSigninRequestArgs = getSigninArgs(includeBillingScope);
-  const authInstance = getAuthInstance();
-  const loadedAuthTokenResponse: User | null = await (popUp
-    ? // returns Promise<User | null>, attempts to use the refresh token to get a new authToken
-      authInstance.signinPopup(args)
-    : authInstance.signinSilent(args)
-  ).catch((e) => {
+  const authInstance: AuthContextProps = getAuthInstance();
+  try {
+    const loadedAuthTokenResponse: OidcUser | null | AuthTokenErrorState = await (popUp
```

Comment:
```suggestion
    const loadedAuthTokenResponse: OidcUser | null = await (popUp
```

With the `.catch(...)` removed, this no longer possibly returns an `AuthTokenErrorState`, right?

Reply:
Good catch, I will correct that 👍